### PR TITLE
#1532 - Add log system, rabbit queue, and vault health to status API

### PR DESCRIPTION
**Pull request by JohnPTobe**
##### Checklist

- [x] `manage.py test` passes
- [x] tests are included
- [x] documentation is changed or added

### Affected app(s)
- messaging
- scale
- scheduler
- util

### Description of change
Adding status messages for scale dependencies.

### CODE REVIEW

**Comment by JohnPTobe** [comment]
Docs are done; ready for a second pass

File: scale/scale/local_settings_docker.py [hunk 1]
```diff
@@ -33,6 +33,8 @@
 STATIC_ROOT = os.environ.get('SCALE_STATIC_ROOT', 'static/')
 STATIC_URL = os.environ.get('SCALE_STATIC_URL', '/service/%s/static/' % FRAMEWORK_NAME)
 
+FLUENTD_BUFFER_WARN = os.environ.get('FLUENTD_BUFFER_WARN', FLUENTD_BUFFER_WARN)
```
[the remark this answers is not in the document]
Comment (JohnPTobe) [reply scale/scale/local_settings_docker.py]: done

File: scale/scale/settings.py [hunk 2]
```diff
@@ -62,6 +62,10 @@ def get_env_boolean(variable_name, default=False):
 # Directory for rotating metrics storage
 METRICS_DIR = None
 
+# fluentd warning levels, or -1 to disable warnings
+FLUENTD_BUFFER_WARN = -1
```
[the remark this answers is not in the document]
Comment (JohnPTobe) [reply scale/scale/settings.py]: ditto

File: scale/scale/settings.py [hunk 3]
```diff
@@ -75,6 +79,9 @@ def get_env_boolean(variable_name, default=False):
 
 DATABASE_URL = os.getenv('DATABASE_URL')
 
+#root url for scale installation
+SCALE_HOST = os.getenv('SCALE_HOST', 'http://localhost:8000')
```
[the remark this answers is not in the document]
Comment (JohnPTobe) [reply scale/scale/settings.py]: switched to vhost

File: scale/scheduler/dependencies/manager.py [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+    def _generate_idam_status(self):
+        """Generates the IDAM (GEOAxIS) status message
+
+        :return: JSON describing the IDAM status
+        :rtype: dict
+        """
+
+        status_dict =  {'OK': False, 'detail': {}, 'errors': [], 'warnings': []}
+        if not scale_settings.GEOAXIS_ENABLED:
+            status_dict = {'OK': True, 'detail': {'geoaxis_enabled': False, 'msg': 'Geoaxis is not enabled'}, 'errors': [], 'warnings': []}
```
Comment (emimaesmith) [inline scale/scheduler/dependencies/manager.py]: GEOAxIS

**Comment by JohnPTobe** [comment]
All fixed :)

**Pull request merged**